ideascube/search/utils.py

```diff
@@ -12,7 +12,7 @@ def create_index_table(force=True):
         cursor_transient.execute("DROP TABLE IF EXISTS idx")
         cursor_transient.execute("CREATE VIRTUAL TABLE idx using "
                                  "FTS4(id, model, model_id, public, text, "
-                                 "lang, kind, tags)")
+                                 "lang, kind, tags, package_id)")
```

Comment:
So: being able to combine other filters with the package_id is very nice. But, adding more column to the Search model does not seem future proof to me.
Also, I think that that a given media or a media is part of a package is only internal business, and I feel like user will get what a package is.
So my suggestion is to use tags for that: when creating a media from a package, we also add its *package name* as tag. And then when linking from a package card in the home page we just use this tag, which will appear as a tag on the filters and so on.
Of course, we may face the case where two packages have the same name (while not the same id), or that someone add a package name as tag to a media not related to this package. I think that's acceptable, and *may* also be considered as a feature :)